codecov_cli/services/commit/base_picking.py

```diff
@@ -14,7 +15,7 @@ def base_picking_logic(base_sha, pr, slug, token, service):
         "user_provided_base_sha": base_sha,
     }
     headers = get_token_header_or_fail(token)
-    url = f"https://api.codecov.io/api/v1/{service}/{slug}/pulls/{pr}"
+    url = f"{CODECOV_API_URL}/api/v1/{service}/{slug}/pulls/{pr}"
```

Comment:
Shouldn't enterprise customers also be able to pick their base?...

Reply:
you're totally right